Update tag.mdx (#3061)
grommet/hpe-design-system: aries-site/src/pages/components/tag.mdx

Name-value pairing with tags allow further organization and identification of resources. While this feature is not a requirement, name-value pairs can help users more clearly group resources.

> - In some hybrid use cases it can be valid to define either just the value or a combination of name-value by making the value field optional.

Reviewer: ```suggestion
- In some hybrid use cases, it can be valid to have a mixture of tags with name-value and others with just value.
```
wondering if this better captures the design desire?

Reviewer: Seems like the "name" is the optional element?

Author: @taysea that better captures the hybrid approach. Good catch, yes the 'name' is the optional element.

  <CreateTag />
</Example>

> ## Add hybrid example HERE

Reviewer: ```suggestion
```

Reviewer: suggesting we omit the placeholder for now.
